### PR TITLE
Cb new upload

### PR DESCRIPTION
**Pull request by ncrouzier-nist**
Pull my gvt change in this branch. It looks fine but might want tot keep it on a seperate branch until we're sure.
Also check the hit-core and hit-core-hl7v2 changes

### CODE REVIEW

File: hit-base-web/src/main/webapp/views/logs/validation.html [hunk 1]
```diff
@@ -11,12 +11,15 @@
 </script>
 <div ng-controller="ValidationLogCtrl" ng-init="initValidationLogs()">
 
+<<<<<<< HEAD
```
Comment (haffo) [inline hit-base-web/src/main/webapp/views/logs/validation.html]: Conflicts issue.Can you fix

File: hit-base-web/src/main/webapp/views/profileverification.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<style id="antiClickjack">
```
Comment (haffo) [inline hit-base-web/src/main/webapp/views/profileverification.html]: Isn't this specific to GVT ? Not sure we want it here. Any thought ?

File: hit-base-service/pom.xml [hunk 3]
```diff
@@ -1,16 +1,16 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<artifactId>hit-base-service</artifactId>
```
Comment (haffo) [inline hit-base-service/pom.xml]: Shouldn't the name stay hit-base-service ?

File: hit-base-web/src/main/webapp/views/profileverification.html (hunk 2 above)
Comment (ncrouzier-nist) [reply hit-base-web/src/main/webapp/views/profileverification.html]: yes correct will remove tomorrow

File: hit-base-service/pom.xml (hunk 3 above)
Comment (ncrouzier-nist) [reply hit-base-service/pom.xml]: Yes will fix.

File: hit-base-web/src/main/webapp/views/logs/validation.html (hunk 1 above)
Comment (ncrouzier-nist) [reply hit-base-web/src/main/webapp/views/logs/validation.html]: will do

**Review by haffo — APPROVED** [review]

**Pull request merged**